lib/identity_cache/cache_hash.rb

```diff
@@ -1,16 +1,23 @@
-# Use CityHash for fast hashing if it is available; use Digest::MD5 otherwise
-begin
-  require 'cityhash'
-rescue LoadError
+# Use a hashing library for fast hashing if it is available; use Digest::MD5 otherwise
+AVAILABLE_HASHING_GEMS = %w{xxhash murmurhash3 cityhash}
+
+hash_loaded = false
+AVAILABLE_HASHING_GEMS.each do |gem|
```

Comment:
> crazy reason you need to add xxhash, it would invalidate the caches without you knowing ?

or more likely, you add some unrelated gem and it indirectly pulls in a dependancy like that.